features/confidential-transactions/index.md

```diff
@@ -90,13 +90,11 @@ elements-cli sendrawtransaction $TX
 
 The implementation of Confidential Transactions as it appears in Elements has some important limitations to be aware of.
 
-The implementation only hides a certain number of the digits of the amount of each transaction output, dependent on the range proof's "blinding coverage" at a desired precision level.  Subsequently, there is a "minimum confidential amount" around 0.0001, and a "maximum confidential amount" that is 2<sup>32</sup> times the minimum amount.
-
-Digits smaller than the minimum will be revealed to observers; for example, if the minimum is 0.0001, a transaction output of 123.456789 will look like "?.????89". The tiny amount of information about the value leaked in this way is unlikely to be important, but be aware that this could allow observers to "follow" coins by linking subsequent transactions with identical amounts. All values can be rounded to the minimum to avoid revealing information in this way if preferred.
+The implementation only hides a certain number of the digits of the amount of each transaction output, dependent on the range proof's "blinding coverage" at a desired precision level. The default blinding precision is 52 bits, which will cover any value up to 2^52 satoshi. The default equates to a maximum amount of just over 45 million. For assets that are pegged to Bitcoin, such as L-BTC on the Liquid network, this provides cover for all possible pegged asset transactions. It should be noted that for Issued Assets it is possible to exceed the default maximum confidential amount by repeatedly reissuing the asset and then spending an amount greater than the limit in a single transaction.
```

Comment:
You cannot have output with nValue > MAX_MONEY, and MAX_MONEY in Elements is the same as in Bitcoin Core, so it seems that the last sentence does not apply anymore for ct_bits=52

Reply:
You cannot have an unconfidential output with that amount. Confidential likely works(wallet might get confused?)

Reply:
Or rather it applies for values of CT_BITS < 51

Reply:
Right, confidential amounts cannot be checked for MAX_MONEY.

By the way, what happens when I issue asset with confidential amount 2^64-1 and then reissue it ? I can have total asset supply more than 2^64 ?

Reply:
You wouldn't be able to create a rangeproof for a single output of more than 64 bits of hiding, IIUC

Reply:
Right, first 64 bits in the output of an issuance, and another 64 bits in the output of a reissuance (and another 64 bits in the second reissuance...)

Reply:
> (wallet might get confused?)

before that, UnblindConfidentialPair will fail: https://github.com/ElementsProject/elements/blob/3c029f42bdb763c6b29512fdbdaef6c135708f2e/src/blind.cpp#L93

So the wallet will store that UTXO with value = -1 and ignore it